docs/theory/statistics.rst

```diff
@@ -98,6 +98,15 @@ quantities typically have multiplicative errors. In other words, if the
 measured value is large, then the associated error is also proportionally
 large.
 
+Summary statistics
+------------------
+
+It is common for experimental results to be reported in the form of a sample
+mean and standard deviation. It is important to note that for non-negative
+quantities like metabolite and enzyme concentrations these summary statistics
+will generally not be good values use as :math:`y` and :math:`\sigma` above. If
+possible, non-summarised measurement results should be used instead.
+
```

Comment:
Or unless they're summarised using the assumed measurement model (e.g. lognormal for metabolites)? To do it properly we could have calibration data, measuring a known amount 5 times, to get an estimate of the machine error.

Reply:
Using summary statistics that work under the assumed measurement model would definitely work but I haven't worked out the details. For now I think it's ok to leave that out and give a recommendation that is definitely ok.